packages/app/src/systems/Core/utils/address.ts

```diff
@@ -0,0 +1,3 @@
+export const formatAddress = (address: string) => {
+  return `${address.slice(0, 6)}...${address.slice(-5)}`;
+};
```

Comment:
can we change to 
```
export function shortAddress(address: string = '') {
  return address.length > 10
    ? `${address.slice(0, 6)}...${address.slice(-4)}`
    : address;
}
```

2 benefits:
- keep same name of wallet
- only make short when lenght is > 10